R/UK.R

```diff
@@ -264,7 +264,7 @@ UK <- R6::R6Class("UK", # rename to country name
       }
       # download and link all data into a single data frame
       safe_reader <- safely(csv_reader)
-      csv <- map(csv_links, ~ safe_reader(.)[[1]], verbose = self$verbose)
+      csv <- map(csv_links, (~ safe_reader(., verbose = self$verbose)[[1]]))
```

Comment:
I don't think this change is required.

Reply:
When I ran this locally verbose wasn't being passed to `csv_reader`

Reply:
Hmm well it should be.... 

Well this way is also fine but can drop the brackets around ~ statement.

Reply:
yes sorry that was an artefact of other trials

Reply:
no problem. Its very strange to me that code as written wasn't working....